packages/shared/src/components/sidebar/MyFeedAlert.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement } from 'react';
+import XIcon from '@dailydotdev/shared/icons/x.svg';
+import { Alerts } from '../../graphql/alerts';
+import { Button } from '../buttons/Button';
+import { AlertContainer, AlertCopy, Pointer } from './common';
+
+const alertCopy = {
+  default: `Psst, your feed has a new name! We've already applied your content filters to it. `,
+  manual: `🎉 Your feed is ready! Click here to manage your feed's settings`,
+};
+
+export default function MyFeedAlert({
+  alerts,
+  hideAlert,
+}: {
+  alerts: Alerts;
+  hideAlert: () => unknown;
+}): ReactElement {
+  if (alerts?.filter || (!alerts?.filter && alerts?.myFeed === null)) {
```

Comment:
@sshanzel Where did you base that on?
We should not change anything in relation to the `alerts.filter`.

Please let me know if we need to have a meeting around this, as this will be very crucial.

Reply:
Oh, cool. Seen the PR for the API now. The condition here looks fine.